.gitattributes

```diff
@@ -4,3 +4,14 @@ tests/data/*-win.php eol=crlf
 
 # Don't show bundled libraries as source files in GitHub
 bundle/**/* linguist-vendored
+
+/.github export-ignore
+/tests export-ignore
+/features export-ignore
+/bin export-ignore
```

Comment:
```suggestion
```
I think excluding `/bin` would mean `vendor/bin/wp` would not work when WP CLI is installed via Composer.

https://github.com/wp-cli/wp-cli/blob/03d30d4138d12b4bffd8b507b82e56e129e0523f/composer.json#L54-L57